Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/CommandLine.kt

```diff
@@ -59,10 +59,20 @@ fun getCInteropArguments(): List<OptionDescriptor> {
                     isMultiple = true, delimiter = ",", deprecatedWarning = "Option -h is deprecated. Please use -header."),
             OptionDescriptor(ArgType.String(), HEADER_FILTER_ADDITIONAL_SEARCH_PREFIX, "hfasp",
                     "header file to produce kotlin bindings for", isMultiple = true, delimiter = ","),
-            OptionDescriptor(ArgType.String(), "compilerOpts", "copt",
-                    "additional compiler options", isMultiple = true, delimiter = " "),
-            OptionDescriptor(ArgType.String(), "linkerOpts", "lopt",
-                    "additional linker options", isMultiple = true, delimiter = " "),
+            OptionDescriptor(ArgType.String(), "compilerOpts",
+                    description = "additional compiler options (allows to add several options separated by spaces)",
```

Comment:
Not in this PR, but may make sense to add options matching to compiler ones, i.e. `-compiler-options`.

Reply:
Make these  deprecated and add new the same? Then CompilerOpt and linkerOpt for single value can be deleted6 they aren't released and nobody use them?

Reply:
Yeah.